src/components/DataTable2/readme.md

```
@@ -58,15 +58,15 @@ To disable searchParams of the DataTable2, `disableParams={true}` is used to cha
 />
 ```
 
-To remove footer from the DataTable2, `hideFooter={true}` should be set.
+To remove footer from the DataTable2, `disableFooter={true}` should be set.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)